<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solution_widget</class>
 <widget class="QWidget" name="solution_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>301</width>
     <height>411</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="steps_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>299</width>
      <height>409</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="btn_widget" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>304</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="method_1">
      <property name="text">
       <string>Method1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="method_2">
      <property name="text">
       <string>Method2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="method_3">
      <property name="text">
       <string>Method3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="left_widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>330</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>160</height>
     </size>
    </property>
    <property name="text">
     <string>Image_input</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="latex_tab">
     <attribute name="title">
      <string>Latex</string>
     </attribute>
     <widget class="QTextEdit" name="latex_textedit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ascii_tab">
     <attribute name="title">
      <string>Asciimath</string>
     </attribute>
     <widget class="QTextEdit" name="ascii_textedit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
